google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -701,6 +704,7 @@ public static class Builder
     private String compressorName;
     private String emulatorHost = System.getenv("SPANNER_EMULATOR_HOST");
     private boolean leaderAwareRoutingEnabled = true;
+    private OpenTelemetry openTelemetry = null;
```

Comment:
nit:
```suggestion
    private OpenTelemetry openTelemetry;
```

(null is the default value)